vault/expiration.go

```diff
@@ -949,14 +954,22 @@ func (m *ExpirationManager) lazyRevokeInternal(ctx context.Context, leaseID stri
 
 // should be run on a schedule. something like once a day, maybe once a week
 func (m *ExpirationManager) attemptIrrevocableLeasesRevoke() {
+	m.core.stateLock.RLock()
```

Comment:
I think we now need to use the pattern of a regular job here, and check the quitCh in addition here.

Reply:
https://github.com/openbao/openbao/blob/45983e33b556bd407a3a925c2d8a3ced78eaa8c8/vault/expiration.go

In particular.